download/upgradeRecipe/upgradeRecipe7.adoc

```diff
@@ -2748,3 +2748,35 @@ rule "Language diversity"
         scoreHolder.reward(kcontext);
 end
 ----
+
+== From 7.26.0.Final to 7.27.0.Final
+
+=== `subPilarEnabled` property is deprecated
```

Comment:
Context is missing.
=> `<pillarChangeMoveSelector>` and ...: subPilarEnabled` is now deprecated